config/main.py

```diff
@@ -1852,27 +1853,70 @@ def override_config_table(db, input_config_db, dry_run):
 
     config_db = db.cfgdb
 
+    # Read config from configDB
+    current_config = config_db.get_config()
+    # Serialize to the same format as json input
+    sonic_cfggen.FormatConverter.to_serialized(current_config)
+
+    updated_config = update_config(current_config, config_input)
+
+    yang_enabled = is_yang_config_verification_enabled(config_db)
+    if yang_enabled:
+        # The ConfigMgmt will load YANG and validate
+        # runnning config during initialization.
+        try:
+            cm = ConfigMgmt()
+        except Exception as ex:
+            click.secho("Failed to validate running config. Error: {}".format(ex), fg="magenta")
```

Comment:
This try statement initializes class and loads data, but I don't believe validation is done until you call `validate_config_by_cm` 
Should the error message be about initialization and loading instead of validation?

Reply:
You are right. It only load_data during the initialization. I will add one line for validation.